.gitignore

```diff
@@ -52,7 +52,6 @@ ui/node
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-syndesis/
 
```

Comment:
Is this safe?

Reply:
Without that I couldn't add the java source files inside a syndesis directory, eg.
`.../syndesis/app/connector/support/util/src/main/java/io/syndesis/connector/support/util/ConnectorOptions.java
`
The 'syndesis' package trips this gitignore and the src file cannot be added.

Reply:
@riccardo-forina 
Would you mind considering the above please?

Reply:
`/syndesis` would make it sure that the rule is applied only at the root of the repo I think. Would that work?

Reply:
Anyway, I'm not sure why that ignore is needed, I think it's been added by @paoloantinori because he noticed some temporary stuff being created while building? I dunno

Reply:
Gone with your suggestion. If this doesn't work, we can revisit.